r/R/rzeppelin/R/scalaInterpreter.R

```diff
@@ -0,0 +1,126 @@
+## Scala scripting over TCP/IP
```

Comment:
Could you explain please, this looks quite different from https://github.com/cran/rscala/blob/master/R/scalaInterpreter.R  or is it a different version we are looking at?